tst/EnergyPlus/unit/OutdoorAirUnit.unit.cc

```diff
@@ -338,5 +338,19 @@ namespace EnergyPlus {
 		EXPECT_DOUBLE_EQ( EAFanPower, 75.0 );
 		EXPECT_DOUBLE_EQ( SAFanPower + EAFanPower, OutAirUnit( OAUnitNum ).ElecFanRate );
 
+		// #6173
```

Comment:
We can always pass the blame back for what commit/PR added a block of code, so no need for comments like this.  The code looks good otherwise though, so unless there's a follow-up commit, this is OK.